packages/crypto/src/Bip32/Bip32PrivateKey.ts

```diff
@@ -123,11 +125,12 @@ export class Bip32PrivateKey {
    * This is why deriving the private key should not fail while deriving
    * the public key may fail (if the derivation index is invalid).
    *
+   * NOTE: You must call `Crypto.ready()` at least once before calling this function.
```

Comment:
```suggestion
   * NOTE: You must await `Crypto.ready()` at least once before calling this function.
```

Reply:
done in https://github.com/input-output-hk/cardano-js-sdk/pull/1558/commits/171910859b1703c0958db59676555b1aec52965a